package/src/custom/tests/School/unit-php/modules/Contacts/Student_GradebookTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Sugarcrm\SugarcrmTestsUnit\TestMockHelper;
+use Sugarcrm\Sugarcrm\Util\Uuid;
+
+require_once 'include/SugarQueue/SugarJobQueue.php';
+
+/**
+ * @coversDefaultClass \Students_Gradebook
+ */
+class Student_GradebookTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var a partial mock of SchedulersJob
+     */
+    private $job;
+
+    /**
+     * @var a partial mock of a Sugar Job Queue
+     */
+    private $queue;
+
+    /**
+     * A partial mock of Students_Gradebook
+     */
+    private $sg;
+
+    /**
+     * @var a partial mock of a Student (Contact) with an id, first name, last name, and email
+     */
+    private $student;
+
+    protected function setUp()
+    {
+        \SugarAutoLoader::load('../../../modules/Contacts/Students_Gradebook.php');
```

Comment:
Another one I would just require directly above the class. And you might not need the `../../../`.